app/tasks/SyncBooksWithGoodReadsJob.py

```diff
@@ -55,7 +55,8 @@ def _sync_book_with_goodreads(self, book):
         book.author = decoded_book_data['author']
         book.rating = decoded_book_data['rating']
         book.description = decoded_book_data['description']
-        book.cover_image_url = decoded_book_data['cover_image_url']
+        if book.cover_image_url is None:
```

Comment:
Replace `if book.cover_image_url is None:` by `if (book.cover_image_url is None) or ('goodreads' in book.cover_image_url):